pages/common/pageres.md

```diff
@@ -0,0 +1,29 @@
+# pageres
+
+> Capture screenshots of websites in various resolutions.
+> More information: <https://github.com/sindresorhus/pageres-cli>.
+> More information on API: <https://github.com/sindresorhus/pageres>.
+
+- Basic multi-url, multi-resolution usage:
+
+`pageres {{any_web_page_url}} {{any_web_page_url}} {{1366x768}} {{1600x900}}`
```

Comment:
Perhaps we could use specific examples here, like `https://example.com/` and `https://example2.com/`? The same goes for subsequent examples.